src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SvType.java

```diff
@@ -1,6 +1,7 @@
-package org.broadinstitute.hellbender.tools.spark.sv;
+package org.broadinstitute.hellbender.tools.spark.sv.discovery;
```

Comment:
Understand. But I am going to keep this under discovery for now because no other tools are accessing it. Once the imprecise variants and genotyping tools are in, we can decide where to move it.